src/networkBuilder.ts

```diff
@@ -37,44 +38,52 @@ export async function buildNetwork(context: NetworkContext): Promise<void> {
+            orchestrateOutputPath = context.outputPath;
+            context.outputPath += "/network";
+            context.privacy = true;
+            context.elk = false;
+            context.nodeCount = 3;
+        }
 
-            const commonFilesPath = path.resolve(filesDirPath, "common");
-            const clientFilesPath = path.resolve(filesDirPath, context.clientType);
+        spinner.text = `Installing ` +
+            `${context.clientType === "besu" ? "Besu" : "GoQuorum"} quickstart ` +
+            `to ${context.outputPath}`;
+        spinner.start();
 
-            if (validateDirectoryExists(commonTemplatePath)) {
-                renderTemplateDir(commonTemplatePath, context);
-            }
+        const commonTemplatePath = path.resolve(templatesDirPath, "common");
+        const clientTemplatePath = path.resolve(templatesDirPath, context.clientType);
 
-            if (validateDirectoryExists(clientTemplatePath)) {
-                renderTemplateDir(clientTemplatePath, context);
-            }
+        const commonFilesPath = path.resolve(filesDirPath, "common");
+        const clientFilesPath = path.resolve(filesDirPath, context.clientType);
 
-            if (validateDirectoryExists(commonFilesPath)) {
-                copyFilesDir(commonFilesPath, context);
-            }
+        if (validateDirectoryExists(commonTemplatePath)) {
+            renderTemplateDir(commonTemplatePath, context);
+        }
 
-            if (validateDirectoryExists(clientFilesPath)) {
-                copyFilesDir(clientFilesPath, context);
-            }
+        if (validateDirectoryExists(clientTemplatePath)) {
+            renderTemplateDir(clientTemplatePath, context);
+        }
+
+        if (validateDirectoryExists(commonFilesPath)) {
+            copyFilesDir(commonFilesPath, context);
+        }
+
+        if (validateDirectoryExists(clientFilesPath)) {
+            copyFilesDir(clientFilesPath, context);
         }
 
         await spinner.succeed(`Installation complete.`);
-        console.log();
+
         if (context.orchestrate) {
-            console.log(`To start your test network, run 'npm install' and 'npm start' in the installation directory, '${context.outputPath}'`);
+            console.log();
+            console.log(`To start Orchestrate, run 'npm install' and 'npm start' in the directory, '${orchestrateOutputPath}'`);
+            console.log(`For more information on the Orchestrate, see 'README.md' in the directory, '${orchestrateOutputPath}'`);
         } else {
-            console.log(`To start your test network, run 'run.sh' in the installation directory, '${context.outputPath}'`);
+            console.log();
+            console.log(`To start your test network, run 'run.sh' in the directory, '${context.outputPath}'`);
+            console.log(`For more information on the test network, see 'README.md' in the directory, '${context.outputPath}'`);
```

Comment:
What was this change intended to fix? Have you tested it?

Notice that in the old version orchestrate wasn't getting the files in the `common` directory tree, but now it is. Is this intended/desirable behaviour?

Reply:
Yes, I've been testing this flow several times locally to validate the installation flow behaves as we intended which means, we want the selected test network (Quorum/Besu) to be loaded along with Orchestrate files.